proto/cosmwasm/wasm/v1/types.proto

```diff
@@ -92,8 +76,6 @@ message ContractInfo {
   // Extension is an extension point to store custom metadata within the
   // persistence model.
   google.protobuf.Any extension = 7 [(cosmos_proto.accepts_interface) = "ContractInfoExtension"];
-  // Status is a status of a contract
-  ContractStatus status = 8;
```

Comment:
Don't we need any more? I cannot find it in:
* https://github.com/line/lbm-sdk/blob/57666b816662d314f448c2879a95be5e977ff087/proto/lbm/wasm/v1/types.proto

Reply:
I'll add contract blacklist function through next PR.
This PR only detach custom protobuf from the cosmwasm protobuf.